src/wagtail_ai/prompts.py

```diff
@@ -41,31 +41,40 @@ def as_dict(self) -> dict:
     {
         "label": "AI Correction",
         "description": "Correct grammar and spelling",
-        "prompt": """You are assisting a user in writing content for their website.
-            The user has provided some text (following the colon).
-            Return the provided text but with corrected grammar, spelling and
-            punctuation.
-            Do not add additional punctuation, quotation marks or change any words:""",
+        "prompt": (
+            "You are assisting a user in writing content for their website. "
+            "The user has provided some text (following the colon). "
+            "Return the provided text but with corrected grammar, spelling and punctuation. "
+            "Do not add additional punctuation, quotation marks or change any words:"
+        ),
         "method": "replace",
     },
     {
         "label": "AI Completion",
         "description": "Get help writing more content based on what you've written",
-        "prompt": """You are assisting a user in writing content for their website.
-            The user has provided some initial text (following the colon).
-            Assist the user in writing the remaining content:""",
+        "prompt": (
+            "You are assisting a user in writing content for their website. "
+            "The user has provided some initial text (following the colon). "
+            "Assist the user in writing the remaining content:"
+        ),
         "method": "append",
     },
 ]
 
 
-def get_prompts():
+def get_prompts() -> list[Prompt]:
+    from .models import Prompt as PromptModel  # Avoid circular import
+
     try:
         return [
-            Prompt(**prompt, id=idx)
-            for idx, prompt in enumerate(
-                getattr(settings, "WAGTAIL_AI_PROMPTS", []) or DEFAULT_PROMPTS
+            Prompt(
+                label=prompt.label,
+                description=prompt.description,
+                prompt=prompt.prompt,
+                method=prompt.method,
+                id=idx,
             )
+            for idx, prompt in enumerate(PromptModel.objects.all())  # type: ignore
```

Comment:
**Suggestion:** We should not be using an index from an array to refer to a prompt anymore. It should use an identifier stored on the model.

Reply:
Updated